bot/cogs/error_handler.py

```diff
@@ -31,7 +31,9 @@ def __init__(self, bot: Bot):
         Error handling emits a single error message in the invoking context `ctx` and a log message,
         prioritised as follows:
 
-        1. If the name fails to match a command but matches a tag, the tag is invoked
+        1. If the name fails to match a command:
+            If it matches shh+ or unshh+, the channel is silenced or unsilenced respectively.
+            otherwise if it matches a tag, the tag is invoked
```

Comment:
```suggestion
            * If it matches shh+ or unshh+, the channel is silenced or unsilenced respectively.
              Otherwise, if it matches a tag, the tag is invoked
```